documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,273 @@
+
+
+.Automatic retrieval of CA certificates for the provider's URL in the console
+
+The ability to retrieve CA certificates, which was available in previous versions, has been restored. The `vSphere Verify certificate` option is in the `add-provider` dialog. This option was removed in the transition to the {ocp} console and has now been added to the console. This functionality is also available for {a-rhv-short}, {osp}, and {ocp-name} providers now. link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+
+
+
+
+.Plans that failed to migrate VMs can now be edited
+
+With this update, you can edit plans that have failed to migrate any VMs. Some plans fail or are canceled because of incorrect network and storage mappings. You can now edit these plans until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
+
+
+.Validation rules are now available for OVA
+
+The validation service includes default validation rules for virtual machines from the Open Virtual Appliance (OVA).
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+
+.Deleting a migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources.
+
+* Importer pods
+* Conversion pods
+* Config maps
+* Secrets
+* Failed VMs
+* Data volumes.
+
+To clean up the the temporary resources, you must archive a migration plan before deleting it. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Plans* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
+
+.Migration of virtual machines with encrypted partitions fails during a conversion (vSphere only)
+
+vSphere only: Migrations from {rhv-short} and {osp} do not fail, but the encryption key might be missing on the target {ocp} cluster.
+
+
+.Migration fails during precopy/cutover while performing a snapshot operation on the source VM
+
+Warm migration from {rhv-short} fails if a snapshot operation is triggered and running on the source VM at the same time as the migration is scheduled. The migration does not wait for the snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+
+.Unable to schedule migrated VM with multiple disks to more than one storage class of type `hostPath`
+
+When migrating a VM with multiple disks to more than one storage classes of type `hostPath`, it might happen that a VM cannot be scheduled. Workaround: Use shared storage on the target {ocp} cluster.
+
+
+.Warm migrations do not support guest operating systems
```

Comment:
this needs to say that some guest operating systems are not supported, how about changing it back to how it was in the release notes of 2.5:
```suggestion
.Non-supported guest operating systems in warm migrations
```